lib/ansiblelint/utils.py

```diff
@@ -361,21 +361,40 @@ def _kv_to_dict(v):
     return dict(__ansible_module__=command, __ansible_arguments__=args, **kwargs)
 
 
+def _sanitize_task(task):
+    """Remove internally used data from task arguments."""
```

Comment:
This docstring is a bit misleading because it implies mutating the object while in fact, a new one is returned. 
```suggestion
    """Return a stripped-off task structure compatible with new Ansible.

    This helper takes a copy of the incoming task and drops
    any internally used keys from it.
    """
```

Reply:
@ssbarnea ^